docs-src/cloud/support-service-guarantees.md

```diff
@@ -0,0 +1,19 @@
+---
+id: support-service-guarantees
+title: What are the services offered by Temporal Cloud?
+sidebar_label: Services
+description: We offer four services to Temporal Cloud customers.
+tags:
+  - temporal cloud
+  - support
+---
+
+We offer four services to Temporal Cloud customers. For access to any of these services, please [create a support ticket](/cloud/support) or contact your dedicated account manager.
+
+- **Technical onboarding**: Every Temporal Cloud customer is offered a dedicated onboarding session led by a Developer Success Engineer.
+  This session covers basic setup and Namespace planning and provides an overview of metrics and observability, an overview of development resources, and information you will need in preparation for your go-live.
+- **Design and code reviews**: Our team works with you to review your Workflow design and code to avoid potential issues and optimize your workload to run on Temporal Cloud.
```

Comment:
```suggestion
- **Design and code reviews:** Our team works with you to review your Workflow design and code to avoid potential issues and optimize your workload to run on Temporal Cloud.
```